voltage/channels.py

```diff
@@ -78,7 +78,26 @@ def jump_url(self) -> str:
         return f"https://app.revolt.chat{server_segment}/channel/{self.id}"
 
     def _update(self, data: OnChannelUpdatePayload):
-        raise NotImplementedError
+        if clear := data.get("clear"):
```

Comment:
as smart as this is it will eventually become a pain to debug however since all of this is going to eventually get rewritten it's fine I suppose